### DISCUSSION TITLE
Is there an easy way of testing (for contributing) that a change is properly done including UI?

### DISCUSSION THREAD

**Discussion by Miguelme** [aminalaee/sqladmin#453] [General]
Is there an easy way of testing (for contributing) that a change is properly done including UI? I am asking because I see there are some tests but usually nothing to do manual testing or confirm the news generated layouts. What do you think it's the best approach for that?

**Comment by aminalaee** [comment]
There are a bunch of tests, including UI views here:
https://github.com/aminalaee/sqladmin/blob/main/tests/test_view_sync.py
Like rendering templates and submitting forms, does that help?

**Comment by Miguelme** [comment]
I'm asking something more practical to see how the UI looks because right now I would like to include the possibility of filtering by some of the columns and the code in the backend is easy to test but there is no way to test easily how the FE would look like with a dummy project or something similar
